packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/s3/S3Service.java

```diff
@@ -377,7 +379,7 @@ public S3ObjectListing listObjects(final String bucketName, final String prefix)
 	 * @return The signature of this request
 	 */
 	public static String getSignature(final String bucket, final String key, final String s3id,
-			final String encryptionKey) {
+																		final String encryptionKey) {
```

Comment:
Are these lines weird in the file or is this a github issue? theres a lot of new weird whitespace here and all over